addons/atragmx/functions/fnc_calculate_solution.sqf

```diff
@@ -42,31 +42,8 @@
  * Public: No
  */
 #include "script_component.hpp"
-
-private ["_scopeBaseAngle", "_bulletMass", "_boreHeight", "_airFriction", "_muzzleVelocity", "_temperature", "_barometricPressure", "_relativeHumidity", "_simSteps", "_windSpeed1", "_windSpeed2", "_windDirection", "_inclinationAngle", "_targetSpeed", "_targetRange", "_drag", "_bc", "_dragModel", "_atmosphereModel", "_storeRangeCardData", "_stabilityFactor", "_twistDirection", "_latitude", "_directionOfFire"];
-_scopeBaseAngle     = _this select 0;
-_bulletMass         = _this select 1;
-_boreHeight         = _this select 2;
-_airFriction        = _this select 3;
-_muzzleVelocity     = _this select 4;
-_temperature        = _this select 5;
-_barometricPressure = _this select 6;
-_relativeHumidity   = _this select 7;
-_simSteps           = _this select 8;
-_windSpeed1          = (_this select 9) select 0;
-_windSpeed2          = (_this select 9) select 1;
-_windDirection      = _this select 10;
-_inclinationAngle   = _this select 11;
-_targetSpeed        = _this select 12;
-_targetRange        = _this select 13;
-_bc                 = _this select 14;
-_dragModel          = _this select 15;
-_atmosphereModel    = _this select 16;
-_storeRangeCardData = _this select 17;
-_stabilityFactor    = _this select 18;
-_twistDirection     = _this select 19;
-_latitude           = _this select 20;
-_directionOfFire    = _this select 21;
+params ["_scopeBaseAngle", "_bulletMass", "_boreHeight", "_airFriction", "_muzzleVelocity", "_temperature", "_barometricPressure", "_relativeHumidity", "_simSteps", "_windSpeed",                 "_windDirection", "_inclinationAngle", "_targetSpeed", "_targetRange", "_bc", "_dragModel", "_atmosphereModel", "_storeRangeCardData", "_stabilityFactor", "_twistDirection", "_latitude", "_directionOfFire"];
```

Comment:
I suggest breaking up this very long line. There also is some strange white space in the middle.